tools/copy-webviewer-files.js

```diff
@@ -0,0 +1,13 @@
+const fs = require('fs-extra');
+
+const copyFiles = async () => {
+  try {
+    await fs.copy('./node_modules/@pdftron/webviewer/public', './client/lib');
```

Comment:
`./client/lib` should probably be `./app/lib` as shown in the previous `package.json`.